package/gluon-config-mode-geo-location/src/0400-geo-location.lua.in

```diff
@@ -0,0 +1,136 @@
+return function(form, uci)
+        local pkg_i18n = i18n 'gluon-config-mode-geo-location'
+        local site_i18n = i18n 'gluon-site'
+
+        local site = require 'gluon.site'
```

Comment:
check indentation in this file please, this line and the lines before have a different indentation

Reply:
Oh yes, indeed ... wrong configuration in vim..